ui/components/Expander/index.js

```diff
@@ -1,14 +1,20 @@
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 import classnames from 'classnames';
-
+import { useEffect } from 'react';
 import styles from './Expander.module.scss';
 
-export default function Expander({ summary, children, className, small, open = false, onToggle = () => {} }) {
-
+export default function Expander({
+  summary,
+  children,
+  className,
+  small,
+  open = false,
+  onToggle = () => {},
+}) {
   const [isOpen, setOpen] = useState(open);
   useEffect(() => {
-    setOpen(open);
-  }, [open])
+    setOpen(true);
+  }, []);
```

Comment:
This is going to break the open/close all functionality.

Reply:
What the old code achieved was to allow the parent component to override the open state of this component, and the bug was caused by that parent component doing so a little too indiscriminantly.

It means that the fix for the issue of things suddenly opening is going to be to reduce the amount that _the parent component_ updates the `open` prop, not to disregard changes (because some changes are important).

Specifically, move the code in the parent component which interprets the set of open filters from the selections into a `useEffect(() => {}, [])` so that changes to the selections based on user input do not trigger changes to the open state.